e2e/staticpvc.go

```diff
@@ -285,6 +345,105 @@ func validateRBDStaticMigrationPV(f *framework.Framework, appPath string, isBloc
 	return err
 }
 
+func validateRBDStaticMigrationPVDeletion(f *framework.Framework, appPath string, isBlock bool) error {
```

Comment:
move this to a new file migration.go in e2e?

Reply:
Done